resources/shaders/deferred_geometry_pass.hlsl

```diff
@@ -138,7 +138,7 @@ PS_OUTPUT main_ps(VS_OUTPUT input) : SV_TARGET
 		s0,
 		input.uv);
 
-	if (output_data.alpha == 0)
+	if (output_data.alpha <= 0.5f)
```

Comment:
No feedback but a question instead: This seems like something we want to change using graphics settings no?

Reply:
Mmmmm, kinda...For now it's just to not make alpha masked textures look like pure garbage, but in general this is where translucency would pick up eventually instead of alpha masking